.github/workflows/dispatch.yml

```diff
@@ -0,0 +1,16 @@
+# This triggers the Build workflow in whatwg/whatwg.org for changes here.
+name: Dispatch
+on:
+  push:
+    branches:
+    - master
+jobs:
+  dispatch:
+    name: Dispatch
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: peter-evans/repository-dispatch@v1
+      with:
+        token: ${{ secrets.DISPATCH_TOKEN }}
```

Comment:
This token is a personal access token for @autofoolip, who I've given write access to this repository. That token has the public_repos scope.

Reply:
Actually, using https://github.com/whatbot would be better. I'll do that instead.

Reply:
Updated now.